_posts/2021-05-19-#18418.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: pr
+date: 2021-05-19
+title: "Increase OUTPUT_GROUP_MAX_ENTRIES to 100"
+pr: 18418
+authors: [fjahr]
+components: ["wallet"]
+host: glozow
+status: upcoming
+commit: 241e141
+---
+
+## Notes
+
+* The way our wallet constructs transactions over time can leak information
+  about its contents.  The most obvious example is we can assume that all UTXOs
+to the same address are controlled by the same person. UTXOs sent to different
```

Comment:
"all UTXOs to the same address" -> "all UTXOs sent to the same address"/"all UTXOs with the same address"/"all UTXOs with the same scriptPubKey"